### DISCUSSION TITLE
Contributing to Email Platform Status

### DISCUSSION THREAD

**Discussion by husseinalhammad** [email-markup-consortium/email-markup-consortium#41] [Ideas]
Similar topic to my other thread (#40).

The Email Platform Status was brought up after @M-J-Robbins shared https://web.dev/compat2021/ with us and we highlighted how we don't have feature usage stats in HTML email.

The [Email Platform Status](https://www.emailplatformstatus.com/) is another project by the awesome @hteumeuleu and his team. On Slack he said they would be happy for us to contribute. @avigoldman also mentioned we could aim to pull a bigger data source.

What are our thoughts on contributing to this project?

**Comment by Malvoz** [comment]
> The Email Platform Status

Is that https://www.emailplatformstatus.com/?

**Reply by husseinalhammad** [reply]
Oh of course I forgot to include the link. Yes, that's the one!

**Comment by husseinalhammad** [comment]
Just reporting that during the [last call](https://github.com/email-markup-consortium/email-markup-consortium/blob/main/meeting-notes/2022-02-22.md#20-minutes-summary-of-projects) when going over the summary of projects, the consensus is that this is a project worth pursuing and can potentially help us work towards our goals.

**Comment by M-J-Robbins** [comment]
I've just added a comment on [Environmental impact discussion](https://github.com/email-markup-consortium/email-markup-consortium/discussions/39#discussioncomment-2264693) about maybe looking to collect file sizes here too.

**Comment by M-J-Robbins** [comment]
A couple of idea that we could also include in the Email Platform Status 
* Framework used - this can possibly be identified by certain class names
* Email client targeting used - we can reference [how to target email](https://github.com/dylanatsmith/howtotarget/tree/main/hacks/_posts) and match selectors

**Reply by husseinalhammad** [reply]
Members shared more ideas during the 2022-06-28 meeting:
https://github.com/email-markup-consortium/email-markup-consortium/blob/main/meeting-notes/2022-06-28.md#email-platform-status

> - Hacks (e.g. targeting hacks like those listed in [howtotarget.email](https://howtotarget.email))
> - Frameworks (MJML, Maizzle, etc.)
> - Image formats
> - External resources (e.g. CSS)
> - HTML attributes
> - Expand `@media`
> - Typos and/or invalid features?
>   - if features are checked against valid features, we would be able to see invalid features
